projects/ion/src/lib/chip/chip.component.spec.ts

```diff
@@ -54,6 +60,13 @@ describe('ChipComponent', () => {
     }
   );
 
+  it('should render icon on left', async (iconPosition: Direction = 'left', icon = 'close') => {
+    await sut({ label: 'custom-position', iconPosition, icon });
+    const iconDinamic = screen.queryAllByTestId('icon-dinamic');
```

Comment:
This variable isn't being used, consider removing it.

```suggestion
```

Reply:
thanks for the reviews, i just made the requested changes, do i need to change something else??

Reply:
everything looks good @larissacard, good job!